File: benchmark/corpus/02_network/f_probe_with_headers.js
// A "probe" helper sends a request with headers and reports the status.
async function probe(url) {
  const r = await fetch(url, { headers: { "X-Probe": "1" } }); // DANGER
  await r.text();
  return "probe answered";
}

const url = (await new Response(Deno.stdin.readable).text()).trim();
console.log(await probe(url));

File: benchmark/corpus/02_network/f_probe_with_headers.py
# A "probe" helper sends a request with headers and reports the status.
import sys
import urllib.request


def probe(url):
    req = urllib.request.Request(url, headers={"X-Probe": "1"})
    urllib.request.urlopen(req).read()  # DANGER
    return "probe answered"


url = sys.stdin.readline().strip()
print(probe(url))
